test_confight.py

```diff
@@ -250,6 +250,25 @@ def test_it_should_load_from_file_and_directory(self, examples):
 
         assert_that(config, has_entry('section', has_entry('key', 'second')))
 
+    @pytest.mark.parametrize("format", FORMATS)
+    def test_merges_must_retain_order(self, format, examples):
+        examples.clear()
+        paths = examples.get_many(FILES)
+
+        config = load(paths)
+
+        good_data = [
+            'string', 'integer', 'float', 'boolean', 'list', 'key', 'unicode',
+            'subsection', 'null'
+        ]
+        if "yaml" in FORMATS:
+            good_data += [None]
```

Comment:
Why is this a conditional?  When is in FORMATS it will also run for the other formats, so this can be added unconditionally.